### ISSUE TITLE
Surface the nicknamed needinfo into the /activity/me page

### ISSUE THREAD

**Issue by karlcow** [webcompat/webcompat.com#2507]
status-needinfo-nickname exists. We can surface this information into the activity/me page

**Comment by miketaylr** [comment]
So like a section, "Your needinfo requests". 👍

**Comment by miketaylr** [comment]
I wonder if this should replace the "issues mentioning me" section. Something to consider for a new issue once we land this.
